pkg/metrics/statsd/e2e/metrics.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/Azure/ARO-RP/pkg/metrics"
+)
+
+const (
+	DimensionARMGeoLocation = "armGeoLocation"
+	DimensionARMResourceID  = "armResourceID"
+	DimensionResourceType   = "resourceType"
+	DimensionSucceeded      = "Succeeded"
+
+	LogEntryIsE2EEmittableMetric = "IsE2EEmittableMetric"
+	LogEntryMetricName           = "MetricName"
+	LogEntryMetricStatus         = "MetricStatus"
+)
+
+type E2ELogToMetrics struct {
+	m metrics.Emitter
+}
+
+func NewE2ELogToMetrics(m metrics.Emitter) E2ELogToMetrics {
+	return E2ELogToMetrics{
+		m: m,
+	}
+}
+
+func (e *E2ELogToMetrics) PostMetricsFromLogEntry(entry *logrus.Entry) {
+	if v, ok := entry.Data[LogEntryIsE2EEmittableMetric]; ok {
+		if strings.EqualFold(fmt.Sprint(v), "true") {
```

Comment:
What behavior should we expect to see if either of the checks in these lines fail?

Reply:
If either check fails, then the log entry does not correspond to an e2e test result log entry. Technically the `LogEntryIsE2EEmittableMetric` cannot be set to anything other than `true` if it exists, so the second check can be removed.

Reply:
Removed the second `if` statement.